src/scores/__init__.py

```diff
@@ -1,13 +1,24 @@
 """
 The philosphy is to import the public API during the init phase rather than leaving it to the user
 """
+# pylint: disable=E0603
 
 import scores.categorical
 import scores.continuous
 import scores.functions
 import scores.pandas
 import scores.probability
 import scores.sample_data
-import scores.stats.statistical_tests
+import scores.stats.statistical_tests  # noqa: F401
```

Comment:
If the unit tests use this import, then we don't need the noqa. Probably not essential, so you can keep this noqa if you want.